elastic/kibana
# [Observability] [Exploratory View] limit breakdown to one series

## Summary

Fixes #113421 

Limits breakdowns in exploratory view to a single series.

<img width="434" alt="Screen Shot 2021-10-04 at 11 17 16 PM" src="https://user-images.githubusercontent.com/11356435/135955005-4bc675f3-aee1-4e1d-8358-1b72290dd266.png">

Testing
* Navigate to exploratory view
* Create a single series and apply a breakdown
* Copy that series. Ensure that the second series has "no breakdown" selected in the breakdown dropdown and that the breakdown dropdown is disabled
* Remove the breakdown on the first series
* Ensure that the breakdown dropdown on the second series is now enabled
* Apply a breakdown to the second series, and ensure that the breakdown dropdown on the first series is now disabled
